scripts/deploy-job.sh

```diff
@@ -15,26 +15,7 @@ PAYLOAD='{\"databaseId\": \"11\"}'
 echo "{
   \"requiredParameters\": [
     {
-      \"stackParameterId\": 5,
-      \"value\": \"$JOB\"
-    }
-  ],
-  \"optionalParameters\": [
-    {
-      \"stackParameterId\": 10,
```

Comment:
why did you delete the optional params?

Reply:
I actually didn't. The script, by mistake, had two echo commands and only the second piped its output to the http command.

It should have been addressed in a separate commit but if you don't object I'll include it here rather than reverting it.